content/09-macro.md

```diff
@@ -124,6 +124,10 @@ This kind of reification only works in places where the internal structure expec
 
 Furthermore, a `new` expression can be reified by providing [haxe.macro.TypePath](http://api.haxe.org/haxe/macro/TypePath.html) argument: `new $typePath()`
 
+##### since Haxe 4.3.0
+
+Using the `$` escape modifiers described above is no longer allowed outside of a `macro` expression. This does not limit any applications though: in cases where this was previously allowed, the `$` escape modifier was not necessary. For example: `var x = $v{MacroTools.example()}` should simply become `var x = MacroTools.example()`.
```

Comment:
I don't think I ever heard the term "escape modifier" before, but I trust you that this is what real developers call something like this!

Reply:
Well, the beginning of this section lists "escape mechanisms … triggered by the `$` character"… I think in the land of Scala macros, a `macro` expression would be a "quasiquote", and a `$` escape would be an ["unquote"](https://docs.scala-lang.org/overviews/quasiquotes/lifting.html). In Rust, a `macro` expression is a "quote", putting stuff into it from outside is [interpolation](https://docs.rs/quote/1.0.26/quote/index.html#example). Interpolation might be a good term but we should be consistent. It clashes a tiny bit with string interpolation, but then again it is a very similar concept.